ngraph/core/reference/include/ngraph/runtime/reference/shuffle_channels.hpp

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cfenv>
+#include <cmath>
+#include <numeric>
+#include <vector>
+
+#include "ngraph/runtime/opt_kernel/reshape.hpp"
+#include "ngraph/shape.hpp"
+
+namespace ngraph
+{
+    namespace runtime
+    {
+        namespace reference
+        {
+            void shuffle_channels(const char* arg,
```

Comment:
Do you consider to move this impl to cpp file?

Reply:
Considered and done